ci-operator/step-registry/openshift/e2e/cert-rotation/prepare-for-shutdown/openshift-e2e-cert-rotation-prepare-for-shutdown-commands.sh

```diff
@@ -61,7 +61,11 @@ function wait-for-nodes-to-be-ready {
         sleep 30
       done
     done
-    oc get csr | grep Pending | cut -f1 -d' ' | xargs oc adm certificate approve || true
+    oc get nodes
+    for i in 1 2 3 4 5; do
+      oc get csr | grep Pending | cut -f1 -d' ' | xargs oc adm certificate approve || true
+      sleep 30
+    done
```

Comment:
Usually, we don't know the exact number of certificates required for approving, to set a timeout and try it several times to be more reliable.
```suggestion
    timeout 180s bash -c "while true;do oc get csr | grep Pending | cut -f1 -d' ' | xargs oc adm certificate approve || true;sleep 30s;done"
```